src/twisted/internet/iocpreactor/iocpsupport.py

```diff
@@ -0,0 +1,27 @@
+__all__ = [
+    "CompletionPort",
+    "Event",
+    "accept",
+    "connect",
+    "get_accept_addrs",
+    "have_connectex",
+    "makesockaddr",
+    "maxAddrLen",
+    "recv",
+    "recvfrom",
+    "send",
+]
+
+from twisted_iocpsupport.iocpsupport import (
```

Comment:
can we have all this as `from twisted_iocpsupport import recv` ?

And maybe add a deprecation so that in the future the code will import directly from twisted_iocpsupport and not from this redirection.

Reply:
probably worth doing https://github.com/asottile/reorder_python_imports everywhere in a followup